tests/php/MapFieldTest.php

```diff
@@ -30,7 +31,9 @@ public function testMapField()
         $this->assertRegExp('/\Wdata-field="Location"\W/', $html);
         $this->assertRegExp('/\Wdata-default-srid="3857"\W/', $html);
         $this->assertRegExp('/\Wdata-multi-enabled="1"\W/', $html);
-        $this->assertRegExp('/\Wvalue="SRID=2193;POINT\(5436343 1760120\)"\W/', $html);
+
+        // @TODO This passes with SS44, SS45, but fails with newer versions.  Also is not in the generated output
```

Comment:
Again my brain is not the best with regex, but odd to see this line fail after SilverStripe 4.5